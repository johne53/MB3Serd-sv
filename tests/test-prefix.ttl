@prefix eg: <http://example.org/a/> .

<http://example.org/a/b> a <http://example.org/Thing> .
<http://example.org/a/b/c> a <http://example.org/Thing> .