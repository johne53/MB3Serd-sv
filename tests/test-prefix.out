<http://example.org/a/b> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
<http://example.org/a/b/c> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
